### PR TITLE
[3.0] upgrade: bring repo names in consistent format

### PR DESCRIPTION
**Pull request by MaximilianMeister**
this format is used in the noderepochecks through the repos-cloud.yml
file

### CODE REVIEW

**Comment by jsuchome** [comment]
Note: in forwardport we need to adapt the help text as well

File: crowbar_framework/app/models/api/upgrade.rb [hunk 1]
```diff
@@ -134,7 +134,7 @@ def adminrepocheck
             repos: {}
           }
           ret[:os][:repos][admin_architecture.to_sym] = {
-            missing: ["SUSE Linux Enterprise Server 12 SP2"]
+            missing: ["SUSE-Linux-Enterprise-Server-12-SP2"]
```
Comment (jsuchome) [inline crowbar_framework/app/models/api/upgrade.rb]: Um... should it actually be `SLES12-SP2-Pool` and `SLES12-SP2-Updates`? 
Both of them until we have a way to finding out which one is missing

File: crowbar_framework/app/models/api/upgrade.rb [hunk 2]
```diff
@@ -143,7 +143,7 @@ def adminrepocheck
             repos: {}
           }
           ret[:openstack][:repos][admin_architecture.to_sym] = {
-            missing: ["SUSE OpenStack Cloud 7"]
+            missing: ["SUSE-OpenStack-Cloud-7"]
```
Comment (jsuchome) [inline crowbar_framework/app/models/api/upgrade.rb]: `["SUSE-OpenStack-Cloud-7-Pool", "SUSE-OpenStack-Cloud-7-Updates"]`

@skazi0 , is this what UI expects?

Comment (skazi0) [inline crowbar_framework/app/models/api/upgrade.rb]: UI doesn't currently check `missing` anywhere but int the main functionality it expects "Pool" and "Updates" exactly as in your comment.

**Pull request merged**